tests/conftest.py

```diff
@@ -1,254 +0,0 @@
-import pytest
```

Comment:
We're using explicitly constructed mocks that we pass into the functions under test, and configure in the test methods.